tsconfig.base.json

```diff
@@ -16,9 +16,18 @@
       "@fluentui/react-conformance": ["packages/react-conformance/src/index.ts"],
       "@fluentui/react-utilities": ["packages/react-utilities/src/index.ts"],
       "@fluentui/react-make-styles": ["packages/react-make-styles/src/index.ts"],
+      "@fluentui/make-styles": ["packages/make-styles/src/index.ts"],
+      "@fluentui/react-provider": ["packages/react-provider/src/index.ts"],
+      "@fluentui/react-theme": ["packages/react-theme/src/index.ts"],
+      "@fluentui/react-theme-provider": ["packages/react-theme-provider/src/index.ts"],
       "@fluentui/keyboard-key": ["packages/keyboard-key/src/index.ts"],
-      "@fluentui/react-menu": ["packages/react-menu/src/index.ts"],
-      "@fluentui/react-focus-management": ["packages/react-focus-management/src/index.ts"]
+      "@fluentui/utilities": ["packages/utilities/src/index.ts"],
+      "@fluentui/merge-styles": ["packages/merge-styles/src/index.ts"],
+      "@fluentui/dom-utilities": ["packages/dom-utilities/src/index.ts"],
+      "@fluentui/theme": ["packages/theme/src/index.ts"],
+      "@fluentui/react-window-provider": ["packages/react-window-provider/src/index.ts"],
+      "@fluentui/react-focus-management": ["packages/react-focus-management/src/index.ts"],
+      "@fluentui/react-menu": ["packages/react-menu/src/index.ts"]
```

Comment:
```suggestion
      "@fluentui/react-focus-management": ["packages/react-focus-management/src/index.ts"],
      "@fluentui/react-menu": ["packages/react-menu/src/index.ts"]
```

FYI: As #17197 have been merged (and @ling1726 pointed out), we don't need these packages for convergence, so we can remove these aliases if it will help you somehow 🙂

Reply:
that's great :) wish I got this info sooner to save lot of time

Reply:
I mentioned packages that can be removed in a diff:
- @fluentui/utilities
- @fluentui/merge-styles
- @fluentui/dom-utilities
- @fluentui/theme
- @fluentui/react-window-provider

Sorry, but I don't understand that screenshot, what package do you have on right? Is it `@fluentui/react-make-styles`? I don't see any dependencies from the list above either on `master` or on this branch in it.

https://github.com/microsoft/fluentui/blob/2ca35f1a1755fa5e5931a175ee199f89c7c4329d/packages/react-menu/package.json#L48-L52

https://github.com/microsoft/fluentui/blob/2ca35f1a1755fa5e5931a175ee199f89c7c4329d/packages/react-make-styles/package.json#L37-L40

Reply:
> but I don't understand that screenshot, what package do you have on right? Is it @fluentui/react-make-styles? I don't see any dependencies from the list above either on master or on this branch in it.

yes, sorry I thought that arrow is self explanatory.

those deps are still there (except set-version)

**react-make-styles deps**
![image](https://user-images.githubusercontent.com/1223799/109950353-b5391b00-7cdc-11eb-8cf0-bf7d43e04689.png)